src/Session.php

```diff
@@ -223,6 +224,31 @@ public function toArray(): array
         return $this->contents ?? [];
     }
 
+    public function rewind(): void
+    {
+        reset($this->contents);
```

Comment:
Should throw if not initialized. Should fix the test failure.

```suggestion
        if (!$this->isInitialized()) {
            throw new RuntimeException('Session not initialized');
        }
        
        reset($this->contents);
```